### DISCUSSION TITLE
pestpp-ies stuck?

### DISCUSSION THREAD

**Discussion by hjia1005** [usgs/pestpp#289] [Q&A] [answered]
I am running pestpp-ies, the initial iteration is done and then the window has been like this for nearly half an hour. Does it take very long to calculate the glm factors? I don't think I had this long waiting before...

![image](https://github.com/usgs/pestpp/assets/33449394/0b84eec1-6e76-4042-babb-c8c2d4f03080)

**Comment by hjia1005** [comment]
It's busy working...
![image](https://github.com/usgs/pestpp/assets/33449394/87545eea-6e5a-4e88-81a9-ed44b980aa2d)

**Comment by cnicol-gwlogic** [comment, marked as answer]
Check in the run manager .log file. You'll see it's doing the threaded calcs for the upgrades. Some models / workflows take longer than others, some in my case can take half a day. Sometimes playing with ies_num_theeads can help, but search this got for more info / discussion on that.

**Reply by hjia1005** [reply]
Yes just checked the log file. Thanks Chris.
